File: docs/example_code/Rust/api-v1-rooms-{room_id}-get.rs
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
    
    let room_id = "7b039c76-e656-454e-a97a-85e7490bade4";
    let url = format!(
        "/api/v1/rooms/{}",
        &room_id
    );

    let response = reqwest::blocking::get(&url)?.text()?;
    println!("{:#?}", resp);
    Ok(())
}

File: docs/example_code/js/api-v1-rooms-{room_id}-get.js
"use strict";

// Fetches and prints out the root response

const room_id = `7b039c76-e656-454e-a97a-85e7490bade4`;

const url = `/api/v1/rooms/{room_id}`;

const getResponse = async url => {
    const response = await fetch(url);
    if (!response.ok) {
        throw new Error(
            `An error has occurred; $(response.status)`
        );
    }

    const result = await response.json();
    return result;
}

const data = getResponse(url).catch(error => error.message);
console.log(data);
